subsys/bluetooth/audio/shell/bap_broadcast_assistant.c

```diff
@@ -665,6 +704,80 @@ static int cmd_bap_broadcast_assistant_add_broadcast_id(const struct shell *sh,
 	/* Store results in the `auto_scan` struct */
 	auto_scan.broadcast_id = broadcast_id;
 	memcpy(&auto_scan.subgroup, &subgroup, sizeof(subgroup));
+	memset(auto_scan.broadcast_name, 0, sizeof(auto_scan.broadcast_name));
+	auto_scan.identified_name = false;
+
+	return 0;
+}
+
+static int cmd_bap_broadcast_assistant_add_broadcast_name(const struct shell *sh,
+							size_t argc, char **argv)
```

Comment:
Indentation is off

Reply:
You can use something like `clang-format` to help you. 

Alternatively make sure your editor is following the rules from https://github.com/zephyrproject-rtos/zephyr/blob/main/.editorconfig - Most editors have a plugin for this: https://editorconfig.org/#download

Reply:
OK.